ristretto/snapcraft.yaml

```diff
@@ -19,10 +19,15 @@ parts:
     plugin: autotools
     configflags:
       - --enable-maintainer-mode  # so src/marshal.h gets generated
-  env:
-    plugin: copy
-    files:
-      wrapper: usr/bin/ristretto.wrapper
+    build-packages:
```

Comment:
I always thought it was a part-specific thing, that's the only reason I moved it. Happy to bring it back to the global space if that's better.